### ISSUE TITLE
Create a release for the tablet (x86)

### ISSUE THREAD

**Issue by cgfrost** [almindor/lighthouse#11]
A few people are getting tablets now. If it's straight forward to do a x86 release in harbor would be great.

**Comment by jgibbon** [comment]
I was wondering that as well, so I tried a quick build in the SDK.
Some variable had to be changed to a number in a .cpp to get it compiling (unknown function call and argument somewhere), but then lighthouse ran on the jolla tablet. 
It's not entirely nice, though:
It shows no battery info and insists on the cpu temperature being extremely high – wrong by a factor of 1000?

**Comment by almindor** [comment]
Hmm that's a bit of an odd result. I expected it to either work ok or not
work at all (with GUI problems of course).

Are you able to ssh into your tablet? How confident are you with that? All
the info is read from the /proc folder.

2016-03-18 17:53 GMT-06:00 jgibbon <notifications@github.com>:

> I was wondering that as well, so I tried a quick build in the SDK.
> Some variable had to be changed to a number in a .cpp to get it compiling
> (unknown function call and argument somewhere), but then lighthouse ran on
> the jolla tablet.
> It's not entirely nice, though:
> It shows no battery info and insists on the cpu temperature being
> extremely high – wrong by a factor of 1000?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/almindor/lighthouse/issues/11#issuecomment-198586427>
>



-- 
Feel the power of Opensource.
Feel the power of Free Pascal.

**Comment by jgibbon** [comment]
Yes, ssh is available and used daily, linux user.  
I'd be glad to check some stuff in /proc and report back if you want me to… could you tell me what you like me to "cat" or run?

**Comment by a-dekker** [comment]
The modifications for the tablet to be made are rather small:
For the various battery checks: replace the path /sys/class/power_supply/battery/ to /sys/class/power_supply/dollar_cove_battery/

And the /sys/class/thermal/thermal_zone0/temp result needs to be divided by 1000.

BTW I could compile for the tablet out of the box (no code change needed).

**Comment by almindor** [comment]
I've merged the harbour branch now to master. This will make things easier for contributors and hopefully a-dekker can re-do his pull requests for the tablet and we can fix this.

**Comment by a-dekker** [comment]
third (and hopefully final) pull request is made.

**Comment by a-dekker** [comment]
Just noticed the new version arrived in the Jolla store, including a i486 tablet version.

**Comment by almindor** [comment]
Can someone please test latest master (build manually) and tell me if everything looks good now? I fixed some issues with sizing etc.

**Comment by a-dekker** [comment]
Personally I think the cpu/mem bar in processes got too big, both on the Jolla 1 and tablet, losing a more compact overview. I rather a a size in between the old and new value, but that might be a personal taste.

**Comment by almindor** [comment]
Yeah it's hard to find a good spot. I'll have to see if there's a DPI/realsize way of calculating correct point size for this.

The reason it got bigger is I switched to Theme.<fontsizes> due to people on Nexus phones complaining of the selection being too small for a finger.
